resources/images/config.go

```diff
@@ -32,8 +32,10 @@ import (
 
 var (
 	imageFormats = map[string]Format{
-		".jpg":  JPEG,
+		".jfif": JPEG,
+		".jpe":  JPEG,
 		".jpeg": JPEG,
+		".jpg":  JPEG,
 		".png":  PNG,
```

Comment:
OK, I think I misunderstood the problem. I thought this was a "media type" problem. I see that there are some rooms for improvements in the above, but I suggest that you instead just do the mapping to `jpg` early (inside `FromRemote`).

Reply:
Well, we should also be able to process local files with `.jpe` and `.jfif` extensions, so I'll enable that. But yeah, with image/jpeg, it would nice to say "use the .jpg extension if an extension wasn't provided." So I'll do that too.

Reply:
OK